### DISCUSSION TITLE
VitalSource - Need help

### DISCUSSION THREAD

**Discussion by gguman** [darkreader/darkreader#10788] [Q&A]
Need help! I turned on DR on the site but when I move between page while reading a book then for 1 sec appearing white blank page. It's awful as at night it flash my eyes. Page code:

        <!DOCTYPE html>
        <html xmlns="http://www.w3.org/1999/xhtml">
          <head>
            <title>content</title>
          </head>
          <body style="color: #FFFFFF; overflow: hidden;">
            <div id="page-content"> *some content here*
            </div>
          </body>
        </html>

I've tried to set color throw  Dev tools but no luck: CSS html, body {color: #000000}

**Comment by bershanskiy** [comment]
Hi @gguman, could you please try adding `!important` to your fix? Most likely, this does not work because selector `html, body` is less specific than inline style.

**Comment by bershanskiy** [comment]
Related issue: #10660
